loglist2/logfilter_test.go

```diff
@@ -66,6 +66,29 @@ func TestSelectUsable(t *testing.T) {
 	}
 }
 
+func TestSelectQualified(t *testing.T) {
+	tests := []struct {
+		name string
+		in   LogList
+		want LogList
+	}{
+		{
+			name: "Sample",
+			in:   sampleLogList,
+			want: subLogList(map[string]bool{}),
+		},
+	}
```

Comment:
This would be better if there was at least one qualified log in the log list, otherwise we don't know whether it's actually working.

Reply:
Added.